src/dawn/IIR/MultiStage.h

```diff
@@ -90,6 +90,8 @@ class MultiStage : public IIRNode<Stencil, MultiStage, Stage, impl::StdList> {
   int getID() const { return id_; }
   /// @}
 
+  void setID(int id) { id_ = id; }
```

Comment:
inline

Reply:
why?

https://llvm.org/docs/CodingStandards.html#id52

Reply:
uhmm right, only when it is defined out of the body. Thanks